### PR TITLE
[WIP] Fix CI build/test jobs

### PR DESCRIPTION
**Pull request by michaelosthege**

### CODE REVIEW

**Comment by michaelosthege** [comment]
From https://github.com/conda/conda/issues/9367 it looks like there's a compatibility issue between conda-build and Python 3.8

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -3,63 +3,105 @@ name: Python package
...
-      run: |
-        conda install -c conda-forge conda-build conda-verify
-    - name: Build sundials locally
-      run: |
-        git clone https://github.com/conda-forge/sundials-feedstock
-        conda build -c conda-forge ./sundials-feedstock/recipe
-    - name: Build sunode
+    - uses: actions/checkout@v2
+    - name: Debug dir
+      shell: cmd /C call {0}
```
Comment (michaelosthege) [inline .github/workflows/main.yml:73]: @aseyboldt Where does `{0}` come from? Or is this some kind of shell-switching?

Comment (aseyboldt) [reply .github/workflows/main.yml:73]: That makes sure that we really get a properly activated conda env. See https://github.com/marketplace/actions/setup-miniconda#IMPORTANT
Took me ages to find that...

**Comment by aseyboldt** [comment]
Finally, after 57 commits :-)
@michaelosthege Thanks for your help!

**Comment by michaelosthege** [comment]
Wow! You got it right in the end 💯 I just split the job definition ;)

**Pull request merged**